### DISCUSSION TITLE
Sales details with total Quantity.

### DISCUSSION THREAD

**Discussion by vinitdesai** [opensourcepos/opensourcepos#3904] [Ideas]
It will be helpful if there is total  Quantity displayed on sales page.

**Comment by odiea** [comment]
It is below Customer form.

![2023-12-20 05_34_04-Odies Test _ Powered by OSPOS 3 4 0-dev](https://github.com/opensourcepos/opensourcepos/assets/10962177/8f2e2983-f0cb-4f5d-9275-05766e6d0400)

**Reply by vinitdesai** [reply]
Sorry, i forgot to mention,  it's not there on  SALES RECEIPT page.  it would be helpful to have it there.
